### ISSUE TITLE
Add package but don't write it to disk then reload - walkback

### ISSUE THREAD

**Issue by ericwinger** [GemTalk/Jadeite#183]
- Load RowanSample1 project
- Add Package Test
- Load RowanSample1 project again 
- Get walkback

The bug is trying to update the dirty state. Probably shouldn't try to go to disk if
the package isn't there. 

```
a UserDefinedError occurred (error 2318), reason:halt, No loaded package named 'Test' found
[] in  ExecBlock0 (RwGsImage class) >> loadedPackageNamed: @5 line 5
RwGsImage class >> loadedPackageNamed:ifAbsent: @4 line 11
RwGsImage class >> loadedPackageNamed: @2 line 4
RwPackage >> _loadedPackage @4 line 3
RwPackage >> isDirty @2 line 3
RowanPackageService >> rowanDirty @3 line 3
RowanPackageService >> isDirty @2 line 3
RowanPackageService >> update @12 line 11
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @6 line 11
OrderedCollection (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1

**Comment by ericwinger** [comment]
Note - I've seen multiple types of misbehavior. Sometimes there is no walkback but the unwritten packages are still showing. Probably should erase the package list before a project load.

**Comment by ericwinger** [comment]
Erased package list before loading project. Fixed in 3.0.36.
